centreon/www/front_src/src/Header/Poller/usePollerDatas.ts

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect, useMemo } from 'react';
+
+import { useNavigate } from 'react-router-dom';
+import { useAtomValue } from 'jotai/utils';
+import { useTranslation } from 'react-i18next';
+import { equals, isNil } from 'ramda';
+
+import { useFetchQuery } from '@centreon/ui';
+import { refreshIntervalAtom, userAtom } from '@centreon/ui-context';
+
+import useNavigation from '../../Navigation/useNavigation';
+import { pollerListIssuesEndPoint } from '../api/endpoints';
+import { pollerIssuesDecoder } from '../api/decoders';
+
+import { getPollerPropsAdapter } from './getPollerPropsAdapter';
+import type { GetPollerPropsAdapterResult } from './getPollerPropsAdapter';
```

Comment:
We can discuss about a strategy for where we must define common typings (used across the files)

Reply:
yep, it may be beneficial.